packages/react-components/src/autocomplete/src/Autocomplete.tsx

```diff
@@ -452,7 +452,7 @@ export function InnerAutocomplete(props: InnerAutocompleteProps) {
                         disabled,
                         validationState,
                         fluid,
-                        active,
+                        active, //TODO: TS active doesnt exist on search input
```

Comment:
active doesnt exist on search input, however we pass it in the auto complete. Should i remove the active property from the AutoComplete?

Reply:
@patricklafrance